### ISSUE TITLE
React Build Script Issue - strip-bom 3

### ISSUE THREAD

**Issue by paulkarcher** [Keyang/node-csvtojson#255]
Hello, I am receiving the following error when trying to create a production build for a react application with react-scripts which uses version 1.1.11 of the csvtojson package.
```
$ react-scripts build
Creating an optimized production build...
Failed to compile.

Failed to minify the code from this file: 

        ./node_modules/strip-bom/index.js:2 

Read more here: http://bit.ly/2tRViJ9

error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

I think it's because version 3 of strip-bom uses an ES6 function declaration and the react build script expects the source be using ES5 (as noted [here](https://github.com/facebook/create-react-app/blob/master/packages/react-scripts/template/README.md#npm-run-build-fails-to-minify)). I was able to create a successful build by changing the 2nd line inside the node module source of strip-bom to this:
``` javascript
module.exports = function (x) {
```

I don't know how important react support is for this library, but I figured I would bring it up. This is the commit where the strip-bom version was updated: https://github.com/Keyang/node-csvtojson/commit/ac1b871af413a728ff49af93f5c9086faebc96f2

React applications using react-scripts to build can still create production builds with csvtojson 1.1.9 for now.

**Comment by paulkarcher** [comment]
Here is a link as well to an older discussion on ES5 transpilation in the strip-bom repository: https://github.com/sindresorhus/strip-bom/issues/7

**Comment by Keyang** [comment]
thanks for pointing this out.
released in 1.1.12

**Comment by paulkarcher** [comment]
No problem at all, happy to help.

Looks like that did the trick, thank you for such a quick fix!
